clusterloader2/pkg/test/interface.go

```diff
@@ -34,8 +34,8 @@ type OperationType int
 const (
 	// CREATE_OBJECT is create object operation.
 	CREATE_OBJECT = OperationType(0)
-	// UPDATE_OBJECT is update object operation.
-	UPDATE_OBJECT = OperationType(1)
+	// PATCH_OBJECT is update object (using patch) operation.
+	PATCH_OBJECT = OperationType(1)
```

Comment:
Done.

About adding update: I implemented it some time ago. However, the problem was that to achieve this I had to create custom changes applying function (I haven't found any function capable of doing this in k8s code). So after discussion with Shyam we decided to use patch method instead - patch is used in kubctl so with this implementation we can use well-tested code instead of something custom made.
I will create separate PR with this update function - maybe we can figure out whether it can be change into proper solution.